integ-test/build.gradle

```diff
@@ -230,7 +230,7 @@ task comparisonTest(type: RestIntegTestTask) {
     systemProperty "queries", System.getProperty("queries")
 }
 
-String baseVersion = "2.6.0"
+String baseVersion = "2.7.0"
```

Comment:
This might have to change to maven repo: 
https://aws.oss.sonatype.org/content/repositories/snapshots/org/opensearch/plugin/opensearch-sql-plugin/2.7.0.0-SNAPSHOT/opensearch-sql-plugin-2.7.0.0-20230309.021954-2.zip

Reply:
See this PR: https://github.com/opensearch-project/reporting/pull/663/files

Reply:
done. thanks.